pyOCD/pyDAPAccess/interface/pyusb_backend.py

```diff
@@ -86,6 +86,9 @@ def getAllConnectedInterface():
                 # This can cause an exception to be thrown if the device
                 # is malfunctioning.
                 product = board.product
+            except ValueError:
+                if board.idVendor is 0xd28 and board.idProduct is 0x204:
```

Comment:
So this code is ignoring exceptions that occur on boards with a vid,pid other than 0xd28, 0x204? Can you provide any more details on this? Which VID/PID does Hexiwear have?